src/main/java/org/broadinstitute/hellbender/transformers/IUPACReadTransformer.java

```diff
@@ -0,0 +1,23 @@
+package org.broadinstitute.hellbender.transformers;
+
+import org.broadinstitute.hellbender.utils.BaseUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+public class IUPACReadTransformer implements ReadTransformer {
+    private static final long serialVersionUID = 1L;
+    private boolean strictMode;
+
+    public IUPACReadTransformer() {
+        this.strictMode = false;
```

Comment:
Hmm... I'm tempted to say this should actually be true for the HaplotypeCaller. If the input really has bases that aren't even IUPAC then do we really want to silently allow them?

Reply:
Okay, I left the default as false, but constructed the HC transformer with strict=true.